Rubberduck.Parsing/ComReflection/ComInterface.cs

```diff
@@ -9,11 +9,12 @@
 using FUNCDESC = System.Runtime.InteropServices.ComTypes.FUNCDESC;
 using CALLCONV = System.Runtime.InteropServices.ComTypes.CALLCONV;
 using TYPEFLAGS = System.Runtime.InteropServices.ComTypes.TYPEFLAGS;
+using TYPELIBATTR = System.Runtime.InteropServices.ComTypes.TYPELIBATTR;
 using VARDESC = System.Runtime.InteropServices.ComTypes.VARDESC;
 
 namespace Rubberduck.Parsing.ComReflection
 {
-    [DebuggerDisplay("{Name}")]
+    [DebuggerDisplay("{" + nameof(Name) + "}")]
```

Comment:
Why is this not wrapped in `#If DEBUG` while the other is?

Reply:
The other one is calling `nameof` on a function that is also wrapped in `#if DEBUG`. R# suggested the `nameof` change, and I generally think it's a good idea, but it compiles (and ignores?) the annotation in a release build. It's a little silly in how it's designed, because well, debugger?